jdks/download-jdks.sh

```diff
@@ -62,7 +63,7 @@ function download_jdk {
     then
         echo "<<< Already existing, SKIPPING."
     else
-        curl -L  -s -o downloads/jdk-$1 http://download.oracle.com/otn-pub/java/jdk/$jdk_version-$jdk_build_version/jdk-$jdk_version-$1 --cookie "gpw_e24=http%3A%2F%2Fwww.oracle.com%2F; oraclelicense=accept-securebackup-cookie" #--progress-bar
```

Comment:
I understand why you used -v over -s for debugging purposes/local things, however on travis things like a progress-bar and excessive log spam is not what we want. It doesn't only make things less visible (if it continuosly outputs a progress bar) but also we've had trouble with travis killing our build if you output too many chars (which is why the output of ant build-installers only outputs "."). Fun Fact: If you don't output anything, the build is canceled as well.

I am not sure if I am being overly strict with that, I'll try it locally and see if it's too verbose, but it's a detail we could also fix afterwards actually